client/my-sites/navigation/index.jsx

```diff
@@ -80,10 +81,25 @@ class MySitesNavigation extends Component {
 		);
 	}
 
+	// TODO: Add styles
+	renderGlobalSiteViewSidebar() {
+		return (
+			<GlobalSidebar path={ this.props.path }>
+				<li>
+					<a href="/sites">&lt; { translate( 'All Sites' ) }</a>
```

Comment:
Move to a menu item? There's a dashicon for the &lt too https://developer.wordpress.org/resource/dashicons/#arrow-left-alt2